assets/helpers/checkoutForm/checkoutForm.js

```diff
@@ -78,14 +79,48 @@ export const invalidReason = (form: Object | null): string => {
   }
 };
 
-
 export const formElementIsValid = (formElement: Object | null) => {
   if (formElement && formElement instanceof HTMLFormElement) {
     return formElement.checkValidity();
   }
   return false;
 };
 
+type FlowType = 'opf' | 'npf';
```

Comment:
this term is a bit overloaded, what with Flow types! maybe `OldOrNewFlow` or something